hypha/apply/funds/views.py

```diff
@@ -561,6 +563,56 @@ def get_queryset(self):
     def test_func(self):
         return self.request.user.is_apply_staff or self.request.user.is_reviewer
 
+@method_decorator(staff_required, name='dispatch')
+class ExportSubmissionsByRound(BaseAdminSubmissionsTable):
+
+    def export_submissions(self, round_id):
+        csv_stream = StringIO()
+        writer = csv.writer(csv_stream)
+        header_row,values = [],[]
+        index = 0
+        check = False
+
+        for submission in ApplicationSubmission.objects.filter(round=round_id):
+            for field_id in submission.question_text_field_ids:
+                question_field = submission.serialize(field_id)
+                field_name = question_field['question']
+                field_value = question_field['answer']
+                if field_id not in submission.named_blocks:
+                    header_row.append(field_name) if not check else header_row
+                    values.append(field_value)
+                else:
+                    header_row.insert(index,field_name) if not check else header_row
+                    values.insert(index,field_value)
+                    index = index + 1
+
+            if not check:
+                writer.writerow(header_row)
+                check = True
+
+            writer.writerow(values)
+            values.clear()
+            index = 0
+
+        csv_stream.seek(0)
+        return csv_stream
+
+    def get_queryset(self):
+        try:
+            self.obj = Page.objects.get(pk=self.kwargs.get('pk')).specific
+        except Page.DoesNotExist as exc:
+            raise Http404(_("No Round or Lab found matching the query")) from exc
+
+        if not isinstance(self.obj, (LabBase, RoundBase)):
+            raise Http404(_("No Round or Lab found matching the query"))
+        return super().get_queryset().filter(Q(round=self.obj) | Q(page=self.obj))
+
+    def get(self, request, pk):
+        self.get_queryset()
+        csv_data = self.export_submissions(pk)
+        response = HttpResponse(csv_data.readlines(), content_type="text/csv")
+        response['Content-Disposition'] = 'inline; filename=' + str(self.obj) + '.csv'
```

Comment:
For the filename, let's use a slug or cleaned versin of the `self.obj`. We can also be explicit on the what to use as the filename instead of relying on the `__str__` version of the object.

I would suggest going with something like `submission-[id]-export.csv` as filename format.

Reply:
`submission-[id]-export.csv` can be used when only 1 submission is exported. Since we are exporting by rounds, naming the file by round name is more appropriate. 
I will update the pr with cleaned version of `self.obj`

Reply:
My bad.. i really meant `round-[id]-export.csv`.

Reply:
`round-[id]-export.csv` may mean something to devs but i don't think the admin user have any idea about round-id. Round name would be more meaningful in that case or we can do `round-name_[id]-export.csv`

Reply:
sounds good to me.

Reply:
@theskumar Do you mean this by cleaned version of `self.obj `

![image](https://user-images.githubusercontent.com/111306331/230168195-f5d5d222-444f-49c7-8f73-8258d4ab7594.png)

This is how the file name looks. I tried adding id but the format looked wierd and I think following format is sufficient.
![image](https://user-images.githubusercontent.com/111306331/230171063-033c1d94-971c-4916-b83a-5c1b3147f9c1.png)